src/styles/StyleUtils.js

```diff
@@ -202,6 +202,22 @@ function getBackgroundColorStyle(backgroundColor) {
     };
 }
 
+/**
+ * Converts a color in hexadecimal notation into RGB notation.
+ *
+ * @param {String} hexadecimal A color in hexadecimal notation.
+ * @returns {Array} `null` if the input color is not in hexadecimal notation. Otherwise, the RGB components of the input color.
+ */
+function hexadecimalToRGBComponents(hexadecimal) {
+    const components = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexadecimal);
+
+    if (components !== null) {
+        return _.map(components.slice(1), component => parseInt(component, 16));
+    }
+
+    return null;
```

Comment:
This should be an earlier return like

```
if (!components) {
    return;
}
```

Also, is it necessary to return `null` or is returning `undefined` OK too? I assume `undefined` would be OK, so let's use that.

Reply:
`undefined` is fine as well, yeah.